src/irmin-pack/inode.ml

```diff
@@ -678,8 +692,14 @@ struct
 
       let target :
           type ptr.
-          expected_depth:int -> cache:bool -> ptr layout -> ptr -> ptr t =
-       fun ~expected_depth ~cache layout ->
+          expected_depth:int ->
+          cache:bool ->
+          force:bool ->
```

Comment:
can we add a doc for the `force` argument? i.e. that its only set to false for the proofs